virtcontainers/netns_watcher/netns_watcher.c

```diff
@@ -0,0 +1,1251 @@
+/*
```

Comment:
How about giving it a prefix like `kata` or `vc`? :)

I assume it's intentional that this isn't actually mentioned in the `Makefile` yet too?

Reply:
Yes let's call this `kata-netmon` which stands for "Kata network monitor". And yes, the Makefile will come later. I need to finalize the program first.